sorting/heapSort.php

```diff
@@ -0,0 +1,124 @@
+<?php
+  class Node
+  {
+  private $_i; 
+  
+  public function __construct($key)
+  {
+  $this->_i = $key;
+  }
+  
+  public function getKey()
+  {
+  return $this->_i;
+  }
+  }
+  
+  class Heap
```

Comment:
Please make sure to add tests for this data structure in the `tests/DataStructures` directory